instrumentation/rxjava-2.0/library/src/main/java/io/opentelemetry/instrumentation/rxjava2/TracingAssembly.java

```diff
@@ -0,0 +1,338 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Ah by the way for classes adapted from Armeria (most of them? either that or insane coincidence :P), can you add the original license header under `// Includes work from:` like https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/b78a38f85987736b46faef77b07e2e1e7f9c2cc4/instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/LogSpanDecorator.java#L6 ?

Reply:
👍